tests/common/process_utils/mod.rs

```diff
@@ -167,3 +166,8 @@ pub fn run_process_until_response_matches<F: Fn(Output) -> bool>(
     println!("Success: {}", &command_description);
     Ok(())
 }
+
+pub fn sleep(seconds: u64) {
+    let ten_millis = time::Duration::from_millis(1000 * &seconds);
```

Comment:
Duration has a [`from_secs`](https://doc.rust-lang.org/nightly/std/time/struct.Duration.html#method.from_secs) that does exactly that